File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Declare variable for user input
    int height;

    // Get valid input or re-prompt user
    do
    {
        height = get_int("Set the pyramid height from 1 to 8: ");
    }
    while (height < 1 || height > 8);

    // Print row
    for (int row = 1; row <= height; row++)
    {

        // Print space
        for (int spc = 1; spc <= height - row; spc++)
        {
            printf(" ");
        }

        // Print left-side hash
        for (int hsh = 1; hsh <= row; hsh++)
        {
            printf("#");
        }

        // Print gap

        printf("  ");

        // Print right-side hash
        for (int hsh = 1; hsh <= row; hsh++)
        {
            printf("#");
        }

        // New line
        printf("\n");
    }
}

File: week6/Pset6 - Mario-more/mario.py
height = 0
while height not in range(1, 9):
    try:
        height = int(input("Height (1 to 8): "))
    except ValueError:
        print("That's not an integer!")
for row in range(1, height + 1, 1):
    for space in range(height - row):
        print(" ", end="")
    for hash in range(row):
        print("#", end="")
    print("  ", end="")
    for hash in range(row):
        print("#", end="")
    print("")
